lib/shopify_api/conn_helpers.ex

```diff
@@ -46,21 +60,20 @@ defmodule ShopifyAPI.ConnHelpers do
   end
 
   @doc false
-  def auth_code(conn), do: conn.params["code"]
+  def shop_domain(conn), do: shop_domain_from_header(conn) || conn.params["shop"]
 
   @doc false
-  def assign_app(conn, app_name \\ nil) do
-    case fetch_shopify_app(conn, app_name) do
-      {:ok, app} ->
-        Conn.assign(conn, :app, app)
-
-      :error ->
-        conn
-    end
-  end
+  defp fetch_shopify_shop(conn, shop_domain),
+    do: shop_domain |> ShopServer.get() |> optionally_create_shop(conn)
 
   @doc false
   def assign_shop(conn, shop_domain \\ nil) do
+    shop_domain =
```

Comment:
Same comment as @stewart's on line 42-ish - can we disambiguate `shop_domain` from `shop_domain()`?